<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_programme</class>
 <widget class="QMainWindow" name="gestion_programme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_programme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="affichage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>ajouter </string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom de l'émission :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Temps par h :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Les invités :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Le jour :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lnom_emission">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ltemps">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lidinvite">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ljour">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pB_ajouter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifer">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lidemission">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Idemission :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_emission">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>741</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="detett">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>460</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id emission</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_29">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>460</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>consulter</string>
     </attribute>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>id emission</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lidemission1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="Lid">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> id</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lduree1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>duree</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Voir le PDF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri : </string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ltriidinvite">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>id invite</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_chercher">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>661</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_tri">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="satistique">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>satistique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Affcher </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> arduino :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="actionstop">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>550</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="actionplay">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>550</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>540</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Vidéo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Excel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Excel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
